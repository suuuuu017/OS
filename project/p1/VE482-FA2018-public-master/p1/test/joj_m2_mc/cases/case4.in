WRITELN echo exit | ./mumsh
WRITELN exit
