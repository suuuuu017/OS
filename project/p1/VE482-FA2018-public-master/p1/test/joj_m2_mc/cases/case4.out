mumsh $ mumsh $ exit
mumsh $ exit
